backend/src/main/java/org/example/user/controller/member/UserApiController.java

```diff
@@ -19,6 +26,16 @@ public class UserApiController {
 
 	private final UserService userService;
 
+	@ResponseBody
+	@PatchMapping("/api/v1/user-update")
+	public ResponseEntity<UserDto.UserUpdateResponse> userUpdate(
+		@RequestPart("updateRequest") UserDto.UserUpdateRequest updateRequest,
+		@RequestPart(value = "profileImage", required = false) MultipartFile profileImage) {
```

Comment:
이 부분도 아래와 같이 개행 맞춰 주시면 더 보기 좋을 것 같습니다!

```suggestion
	public ResponseEntity<UserDto.UserUpdateResponse> userUpdate(
		@RequestPart("updateRequest") UserDto.UserUpdateRequest updateRequest,
		@RequestPart(value = "profileImage", required = false) MultipartFile profileImage
    ) {
		String name = SecurityContextHolder.getContext().getAuthentication().getName();
		return ResponseEntity.status(HttpStatus.CREATED)
			.body(userService.updateUser(updateRequest, name, profileImage));
	}
```